### ISSUE TITLE
Readme example not working

### ISSUE THREAD

**Issue by Zaffer** [google-github-actions/auth#97]
### TL;DR

This section seems to be not workings as ${{ steps.auth.outputs.access_token }} = ""
`    - name: Access secret
      run: |-
        curl https://secretmanager.googleapis.com/v1/projects/662974556749/secrets/tester-secret \
          --header "Authorization: Bearer ${{ steps.auth.outputs.access_token }}"
`

### Expected behavior

`
--header "Authorization: Bearer ${{ steps.auth.outputs.access_token }}"
`
this should insert the access token from from the previous step

### Observed behavior

`
--header "Authorization: Bearer ${{ steps.auth.outputs.access_token }}"
`
is resulting in this output in the logs:
`
Run curl https://secretmanager.googleapis.com/v1/projects/***/secrets/*** \
  curl https://secretmanager.googleapis.com/v1/projects/***/secrets/***\
    --header "Authorization: Bearer "
`

### Action YAML

```yaml
`
name: Test Yaml File

on:
  push:
    branches:
      - staging

jobs:
  test:
    name: Test gcloud Auth
    permissions:
      id-token: write
      contents: read
    runs-on: ubuntu-latest
    steps:
    - name: Github Action Manager
      uses: actions/checkout@v2
    - id: auth
      name: Authenticate to Google Cloud 
      uses: google-github-actions/auth@v0
      with:
        workload_identity_provider: projects/***/locations/global/workloadIdentityPools/***/providers/***
        service_account: ***@***.iam.gserviceaccount.com

    # Example of using the token:
    - name: Access secret
      run: |-
        curl https://secretmanager.googleapis.com/v1/projects/***/secrets/*** \
          --header "Authorization: Bearer ${{ steps.auth.outputs.access_token }}"

`
```

**Comment by sethvargo** [comment]
Will be fixed by #98
